File: python3/src/useCase/OrderApprovalRequest.py
class OrderApprovalRequest:
    def set_order_id(self, order_id: int):
        self.order_id = order_id

    def get_order_id(self):
        return self.order_id

    def set_approved(self, approved: bool):
        self.approved = approved

    def is_approved(self):
        return self.approved

File: typescript/src/useCase/OrderApprovalRequest.ts
class OrderApprovalRequest {
  private orderId: number;
  private approved: boolean;

  public  setOrderId(orderId: number): void {
    this.orderId = orderId;
  }

  public getOrderId(): number {
    return this.orderId;
  }

  public setApproved(approved: boolean): void {
    this.approved = approved;
  }

  public isApproved(): boolean{
    return this.approved;
  }
}

export default OrderApprovalRequest;

